qemu/tests/balloon_boot_in_pause.py

```diff
@@ -33,18 +35,18 @@ def memory_check(self, step, changed_mem):
         :param step: the check point string
         :type step: string
         :param changed_mem: ballooned memory in current step(compared with
-        last round of memory, i.e. self.old_mmem, instead of self.ori_mem)
+        last round of memory, i.e. self.pre_mem, instead of self.ori_mem)
         :type changed_mem: int
         :return: memory size get from monitor and guest
         :rtype: tuple
         """
         error_context.context("Check memory status %s" % step, logging.info)
         mmem = self.get_ballooned_memory()
         gmem = self.get_memory_status()
-        if (abs(mmem - self.old_mmem)) != changed_mem or (
-                    self.old_gmem and (
-                            abs(gmem - self.old_gmem) - changed_mem) > 100):
-            self.error_report(step, abs(self.old_mmem - changed_mem),
+        if (abs(mmem - self.pre_mem)) != changed_mem or (
```

Comment:
As we discussed, change_mem could be a negative value(current - pre). So you need remove abs in this function, like memory_check in balloon_check.py.

Reply:
Update balloon_boot_in_pause.py, @huangyum call you to have a check.